osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -144,8 +144,8 @@ public static IEnumerable<GlobalAction> GetGlobalActionsFor(GlobalActionCategory
             new KeyBinding(new[] { InputKey.Control, InputKey.Alt, InputKey.MouseWheelUp }, GlobalAction.EditorIncreaseDistanceSpacing),
             // Framework automatically converts wheel up/down to left/right when shift is held.
             // See https://github.com/ppy/osu-framework/blob/master/osu.Framework/Input/StateChanges/MouseScrollRelativeInput.cs#L37-L38.
-            new KeyBinding(new[] { InputKey.Control, InputKey.Shift, InputKey.MouseWheelRight }, GlobalAction.EditorCyclePreviousBeatSnapDivisor),
-            new KeyBinding(new[] { InputKey.Control, InputKey.Shift, InputKey.MouseWheelLeft }, GlobalAction.EditorCycleNextBeatSnapDivisor),
+            new KeyBinding(new[] { InputKey.Control, InputKey.MouseWheelRight }, GlobalAction.EditorCyclePreviousBeatSnapDivisor),
+            new KeyBinding(new[] { InputKey.Control, InputKey.MouseWheelLeft }, GlobalAction.EditorCycleNextBeatSnapDivisor),
```

Comment:
First of all, see comment preceding - with <kbd>Shift</kbd> removed this should presumably become

```suggestion
            new KeyBinding(new[] { InputKey.Control, InputKey.MouseWheelUp }, GlobalAction.EditorCyclePreviousBeatSnapDivisor),
            new KeyBinding(new[] { InputKey.Control, InputKey.MouseWheelDown }, GlobalAction.EditorCycleNextBeatSnapDivisor),
```

and the comment should be gone.

Secondly, this change won't actually have any effect for anyone who's launched the game before. For these types of changes we've typically bundled realm migrations to change the defaults for users post-facto.

Reply:
Yeah I wasn't sure if we want to migrate existing users. If you think that's best I'll add migrations.

Reply:
Nobody has complained about the past migrations (likely because they were written smart, i.e. user customisations were not overwritten), so I'd say it'd be a good thing to have them.

Reply:
Both issues should be resolved, thanks.